src/untyped_array.rs

```diff
@@ -329,11 +330,11 @@ pub trait PyUntypedArrayMethods<'py>: sealed::Sealed {
     /// # Example
     ///
     /// ```
-    /// use numpy::PyArray3;
+    /// use numpy::{PyArray3, PyUntypedArrayMethods};
```

Comment:
Just because I expect users to do this, could this be `PyArrayMethods` as well or does one have import `PyUntypedArrayMethods`? If so, I think we should consider creating a `prelude` module filled with re-exports?

Reply:
No, we need to import the trait that defines the method. I think a `prelude` would make sense. Should I include it with this PR, or should we do it separately?

Reply:
> Should I include it with this PR, or should we do it separately?

Let's do it separately. This PR is nicely targeted and I see no reason to derail it.